fbw-a32nx/src/systems/fmgc/src/flightplanning/new/plans/performance/FlightPlanPerformanceData.ts

```diff
@@ -6,25 +6,34 @@
 import { MathUtils } from '@flybywiresim/fbw-sdk';
 
 export interface FlightPlanPerformanceData {
+    /** in knots */
```

Comment:
For all of these properties, specify what value they take if not filled in.

Reply:
And also what they are, not just the unit.